google-cloud-tools-plugin/src/com/google/gct/idea/appengine/cloud/AppEngineDeploymentRunConfigurationEditor.java

```diff
@@ -112,7 +118,12 @@ public AppEngineDeploymentRunConfigurationEditor(
     updateJarWarSelector();
     userSpecifiedArtifactFileSelector.setVisible(true);
 
-    appEngineCostWarningLabel.setText(GctBundle.message("appengine.flex.deployment.cost.warning"));
+    appEngineCostWarningLabel.setText(
+        GctBundle.message("appengine.flex.deployment.cost.warning",
+        COST_WARNING_OPEN_TAG,
```

Comment:
LGTM

On Mon, Apr 4, 2016 at 5:59 PM, Etan Shaul <notifications@github.com> wrote:

> In
> google-cloud-tools-plugin/src/com/google/gct/idea/appengine/cloud/AppEngineDeploymentRunConfigurationEditor.java
> <https://github.com/GoogleCloudPlatform/gcloud-intellij/pull/590#discussion_r58456630>
> :
>
> > @@ -112,7 +118,12 @@ public AppEngineDeploymentRunConfigurationEditor(
> >      updateJarWarSelector();
> >      userSpecifiedArtifactFileSelector.setVisible(true);
> >
> > -    appEngineCostWarningLabel.setText(GctBundle.message("appengine.flex.deployment.cost.warning"));
> > +    appEngineCostWarningLabel.setText(
> > +        GctBundle.message("appengine.flex.deployment.cost.warning",
> > +        COST_WARNING_OPEN_TAG,
>
> fixed
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/gcloud-intellij/pull/590/files/e912a795d37fd623d0a9a838cb32699957cce682#r58456630>
>